web/locales/en/messages.ftl

```diff
@@ -642,10 +642,9 @@ about-playbook-how-localize-content-5 = Watch our video explainer to help
 
 ## How to add sentences
 about-playbook-how-add-sentences = How do I add sentences?
-about-playbook-how-add-sentences-content-1 = <strong>Sentence Collector</strong> is a tool for collecting and validating public domain sentences. You will <scAccountLink>need to create an account</scAccountLink>. Then, add your language to your <strong>Profile</strong>.  You can <strong>Add</strong> sentences or <strong>Review</strong> previously-added sentences.
+about-playbook-how-add-sentences-content-1 = You can add sentences on the <writePageLink>Write page</writePageLink> or review sentences on the <reviewPageLink>Review page</reviewPageLink>.
```

Comment:
@jessicarose This is an initial draft of this text about contributing sentences to Common voice. Is there a better way to rewrite this? I guess there is 😄

Reply:
This looks great to me.